ddl/ddl_db_test.go

```diff
@@ -767,6 +767,16 @@ func (s *testDBSuite) TestIssue2293(c *C) {
 	s.tk.MustQuery("select * from t_issue_2293").Check(testkit.Rows("1"))
 }
 
+func (s *testDBSuite) TestIssue6101(c *C) {
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.tk.MustExec("use " + s.schemaName)
+	s.tk.MustExec("set sql_mode=''")
+	s.tk.MustExec("create table t1 (quantity decimal(2) unsigned);")
+	s.tk.MustExec("insert into t1 values (500), (-500), (~0), (-1);")
```

Comment:
Are u sure you run this insert query against mysql? 

I am confused because when I tried to, mysql give me an error.

```
mysql> insert into t1 values (500), (-500), (~0), (-1);
ERROR 1264 (22003): Out of range value for column 'quantity' at row 1
```

Reply:
FYI, my mysql version is `Welcome to the MySQL monitor.  Commands end with ; or \g.
Your MySQL connection id is 9
Server version: 5.7.21 Homebrew`

Reply:
My bad, I forget to set sql_mode.